common/component_interface_utils/include/component_interface_utils/rclcpp/service_server.hpp

```diff
@@ -46,8 +62,17 @@ class Service
 #ifdef ROS_DISTRO_GALACTIC
       using rosidl_generator_traits::to_yaml;
 #endif
+      // If the response has status, convert it from the exception.
       RCLCPP_INFO_STREAM(logger, "service call: " << SpecT::name << "\n" << to_yaml(*request));
-      callback(request, response);
+      if constexpr (!has_status_type<typename SpecT::Service::Response>::value) {
```

Comment:
What types have status and what types don't have?
Could you show me some examples? :pray:

Reply:
The [InterfaceVersion](https://github.com/autowarefoundation/autoware.universe/blob/main/common/autoware_ad_api_msgs/interface/srv/InterfaceVersion.srv) is, and this utility can be used for messages other than API.